common/button/button-fr.html

```diff
@@ -0,0 +1,428 @@
+	<li><a href="#disabledstate">État de désactivation</a></li>
+	<li><a href="#bghorizontal">Groupes de Bouton (Horizontale)</a></li>
+	<li><a href="#bgvertical">Groupes de Bouton (Verticale)</a></li>
+	<li><a href="#buttontoolbar">Barres à outils</a></li>
+	<li><a href="#inputgroups">Groupes d'entrée</a></li>
+	<li><a href="#datatoggle">Boutons bascule de données</a></li>
+</ul>
+
+<h2>Bouton contre liens</h2>
+<dl class="dl-horizontal">
+<dt>Bouton</dt>
+<dd>Un bouton effectuera une action sur la page actuelle sans déplacer le focus ou sans éloigner l'utilisateur</dd>
+<dt>Lien</dt>
+<dd>Un lien enverra l'utilisateur à un emplacement, dans la même page via un lien d'ancrage (`#`) ou vers une autre page sans envoyer de données.</dd>
+</dl>
+<p>N'utilisez pas le bouton de rôle sur une ancre, utilisez plutôt un élément de bouton.</p>
+
+<h2 id="basic">Utilisation de base</h2>
+
+<h3>Par défaut</h3>
+
+<h4>Exemple de travail</h4>
+<h5>Échafaudage (par défaut):</h5>
+<button type="submit">Bouton</button>
+<input type="button" value="Entrer" />
+<input type="submit" value="Soumettre" />
+<a href="#" >Lien</a>
+
+<h5>Avec le style par défaut:</h5>
+<button class="btn btn-default" type="submit">Bouton</button>
+<input class="btn btn-default" type="button" value="Entrer" />
+<input class="btn btn-default" type="submit" value="Soumettre" />
+<a class="btn btn-default" href="#" role="button">Lien</a>
+
+<h4>Code</h4>
+<pre><code>
+	// Bouton par défaut:
+	&lt;button type="submit"&gt;
+	Bouton
+	&lt;/button&gt;
+	&lt;input type="button" value="Entrer"/&gt;
+	&lt;input type="submit" value="Soumettre"/&gt;
+	&lt;a href="#" role="button"&gt;
+	Link
+	&lt;/a&gt;
+
+	// Bouton stylisé:
+	&lt;button class="<strong>btn btn-default</strong>" type="submit"&gt;
+	Bouton
+	&lt;/button&gt;
+	&lt;input class="<strong>btn btn-default</strong>" type="button" value="Entrer"/&gt;
+	&lt;input class="<strong>btn btn-default</strong>" type="submit" value="Soumettre"/&gt;
+	&lt;a class="<strong>btn btn-default</strong>" href="#" role="button"&gt;
+	Lien
+	&lt;/a&gt;
+</code></pre>
+
+<h2 id="enhanced">Utilisation améliorée</h2>
+
+<h3 id="colours/styles">Couleurs/Styles</h3>
+<p>Les couleurs <strong>signifient</strong> diverses choses telles qu'un lien vers des informations d'avertissement (bouton de style attention). Le texte de ce bouton stylisé contiendra également un mot décrivant l'état comme "Avis d'avertissement".</p>
+<h4>Exemple de travail</h4>
+<button type="button" class="btn btn-default">Par défaut</button>
+<button type="button" class="btn btn-primary">Primaire</button>
+<button type="button" class="btn btn-success">Succès</button>
+<button type="button" class="btn btn-info">Info</button>
+<button type="button" class="btn btn-warning">Avertissement</button>
+<button type="button" class="btn btn-danger">Danger</button>
+<button type="button" class="btn btn-link">Lien</button>
+<a href="#" class="btn btn-default">Lien sous forme de bouton</a>
```

Comment:
Use the word "ancre" to avoid confusion with the button styled as a link

Reply:
Changes made in the commit "Updated working examples".